src/color.c

```diff
@@ -607,6 +612,75 @@ static int parse_gnu_ls_colors(struct colors *colors, const char *ls_colors) {
 	return ret;
 }
 
+#if __FreeBSD__
+/** Parse the FreeBSD $LSCOLORS format. */
+static int parse_bsd_ls_colors(struct colors *colors, const char *lscolors) {
+	int ret = -1;
+	static const dchar *keys[LSCOLORS_KEYS] = {
+		"di", "ln", "so", "pi", "ex", "bd", "cd", "su", "sg", "tw", "ow"
+	};
+
+	static const char default_colors[LSCOLORS_LEN] = "exfxcxdxbxegedabagacad";
+
+	static const char *fg_codes[256] = {
```

Comment:
Let's add a comment linking to https://man.freebsd.org/cgi/man.cgi?ls(1)#ENVIRONMENT here